libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -265,6 +269,17 @@ float AP_BattMonitor::voltage(uint8_t instance) const
     }
 }
 
+/// get voltage with sag removed (based on battery current draw and resistance)
```

Comment:
This description should note that the current pack voltage is returned rather then the compensated one if the user has selected it. The current description implies to me that it can only fetch the compensated voltage, which isn't correct.